BackendTaskPilot/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*"
```

Comment:
This is a really big security risk, we should not allow just anyone to contact our API. This is even more important here than most API because we have no client validation and we are using a third party API. We should limit this to only our trusted internal services.